### PR TITLE
Rename second semester modules (GRA/GAD)

### PR DESCRIPTION
**Pull request by LadnerJonas**
blue: this PR / red: current state
image

This PR renames:
- Grundlagen: Algorithmen und Datenstrukturen to **GAD** from G: AD 
- Grundlagenpraktikum: Rechnerarchitektur to **GRA** from Gpraktikum: Rechnerarchitektur

Due to these changes the calendar is way more readable, especially on mobile devices.

### CODE REVIEW

File: src/courses.json [hunk 1]
```diff
@@ -29,8 +29,9 @@
   "Einführung in die Informatik ": "INFO",
   "Praktikum: Grundlagen der Programmierung": "PGP",
   "Einführung in die Rechnerarchitektur": "ERA",
+  "Grundlagenpraktikum: Rechnerarchitektur": "GRA",
```
Comment (CommanderStorm) [inline src/courses.json:32]: Is this the ERA-Praktikum in the second semester? If yes this should be named ERA-P imo

File: src/courses.json [hunk 2]
```diff
@@ -29,8 +29,9 @@
...
   "Einführung in die Softwaretechnik": "EIST",
-  "Algorithmen und Datenstrukturen": "AD",
+  "Grundlagen: Algorithmen und Datenstrukturen": "GAD",
```
Comment (CommanderStorm) [inline src/courses.json:34]: This change Breaks the other Algorithm courses:
image

```suggestion
  "Grundlagen: Algorithmen und Datenstrukturen": "GAD",
  "Effiziente Algorithmen und Datenstrukturen": "EAD",
```

**Review by CommanderStorm — CHANGES_REQUESTED** [review]

File: src/courses.json (hunk 1 above)
Comment (LadnerJonas) [reply src/courses.json:32]: Yeah, that is the ERA-Praktikum.
I just wanted to keep it aligned with the official website of the course: https://gra.caps.in.tum.de/login

But ERA-P is fine for me aswell. You can decide :)

File: src/courses.json (hunk 2 above)
Comment (LadnerJonas) [reply src/courses.json:34]: Great catch, fixed with [052f8d4](https://github.com/TUM-Dev/CalendarProxy/pull/98/commits/052f8d40ab0af8291ce13583e86c1c31f2c0776f)

**Review by kordianbruck — APPROVED** [review]

**Pull request merged**